src/components/common/Header/index.tsx

```diff
@@ -53,6 +55,7 @@ function Header({ setIsLoading }: HeaderProps) {
       resetLoggedInState();
       //   await wallet.disconnect();
       setIsLoggingOut(false);
+      if(location.pathname.includes("/my-space")) navigate("/");
```

Comment:
do ` navigate(-1)` instead maybe?

Reply:
the -1 path could be another tab on my-space, added "/" cause the login modal pops up right as you click logout if you are on anywhere in my-space